components/post_view/post_list_row/post_list_row.tsx

```diff
@@ -173,8 +177,9 @@ export default class PostListRow extends React.PureComponent<PostListRowProps> {
         }
 
         return (
-            <Post
-                postId={listId}
+            <PostComponent
+                post={this.props.post}
+                location={Locations.CENTER}
```

Comment:
Is `PostListRow` used in the RHS at all, or does that use `ThreadViewerRow` nowadays?

Reply:
You are right, this might be one of the components that's removed in the removal PR :P